packages/atti-components/rollup.config.js

```diff
@@ -40,7 +40,7 @@ const getPlugins = (minify = true) => [
     'process.env.NODE_ENV': JSON.stringify(NODE_ENV),
   }),
   babel({
-    exclude: 'node_modules/**',
+    exclude: ['node_modules/**', '../../node_modules/**'],
```

Comment:
Just a thought here... does it work if you use a regex like `exclude: \/node_modules\/` ? So you don't have to use relative paths

Reply:
Seems to work :)